index.html

```diff
@@ -0,0 +1,186 @@
+                        </figcaption>
+                        
+                        <figcaption>
+                             <h4>Thumbnail label</h4>
+                            <p>Teste programador </p>
+                        </figcaption>
+                        
+                        <a href="images/Portfolio02.png"><img src="images/Portfolio02.png" width="100"/></a>
+                        
+                        <figcaption>
+                            <h4>Thumbnail label</h4>
+                            <p>Teste programador </p>
+                        </figcaption>
+                        
+                        <a href="images/Portfolio03.png"><img src="images/Portfolio03.png" width="100"/></a>
+                        
+                        <figcaption>
+                            <h4>Thumbnail label</h4>
+                            <p>Teste programador </p>
+                        </figcaption>
+                                                 
+                    </figure>
+                </section>
+
+                <section class="sobremim">
+                    <h2>Quem eu sou</h2>
+                    <p>Hoje me desafio a aprender e desenvolver habilidades para ser destaque no mercado. </p>
+                    <hr>
+                    
+                    <figure>
+                        <img src="images/photo.png" width="50"/>
+                        <figcaption>Joana D'arc</figcaption>
+                    </figure>
+                    
+                    <figure>
+                        <a href="home.html"><img src="images/icon-facebook.png" alt="facebook" width="50"/></a>
+                        
+                        <a href="home.html"><img src="images/icon-instagram.png" alt="instragram" width="50"/></a>
+                        
+                        <a href="home.html"><img src="images/icon-linkedin.png" alt="linkedin" width="50"/></a>
+                        
+                    </figure>
+                    
+                    <h3>Sobre mim</h3>
+                    <p>Hoje me desafio a aprender e desenvolver habilidades para ser destaque no mercado. </p>
+                    
+                    <h3>Habilidades</h3>
+                    <!-- a barra de progresso será mostrada no css e/ou javascript -->
+                                    
+                    <div class="progresso">Java Script</div>
+                    <div class="progresso">
+                    HTML e CSS</div>
+                    <div class="progresso">
+                    jQuery</div>
+                    <div class="progresso">Wordpress</div>
+                    
+                </section>
+
+                <section class="contato">
+                    <h2>Contato</h2>
+                    <p>Hoje me desafio a aprender e desenvolver habilidades para ser destaque no mercado. </p>
+                    <hr>
+                    <div id="email">nayaracn@outlook.com</div>
```

Comment:
Você pode facilitar a vida do teu usuário colocando seu email clicável.

```suggestion
                    <a href="mailto:nayaracn@outlook.com">nayaracn@outlook.com</a>
```